sys/include/bit.h

```diff
@@ -7,11 +7,11 @@
  */
 
 /**
- * @ingroup     cpu_cortexm_common
+ * @ingroup     sys
  * @{
  *
  * @file
- * @brief       Bit access macros for Cortex-M based CPUs
+ * @brief       Bit access macros with bit-banding support for Cortex-M based CPUs
```

Comment:
Some documentation and macros have specific `cortex-m` refernces.

Reply:
So are you sure its not cortex-m specific?

Reply:
The fallback functions are just normal bit operations.
But Cortex-M can have hardware support for that, mapping individual bits to byte locations in a separate RAM region.

Reply:
Ah Ok, but I still think the comment at
https://github.com/RIOT-OS/RIOT/blob/f85ef6f841cbcfa247beed8eb88fb4d811d701e9/sys/include/bit.h#L35
should be fixed then.

Reply:
Is that what you had in mind?

Reply:
yep :)